src/agenda/index.tsx

```diff
@@ -318,9 +322,11 @@ export default class Agenda extends Component<AgendaProps, State> {
   };
 
   onDayChange = (day: XDate) => {
+    if (this.props.disableCalendar) return;
```

Comment:
Seems like it should be called `disableDayChange`... Also, it should disable the touch events as well (meaning the press indication), otherwise it will look buggy